runtime/dancebox/tests/integration_test.rs

```diff
@@ -1791,7 +1791,7 @@ fn test_author_noting_not_self_para() {
                 Some(ContainerChainBlockInfo {
                     block_number: 1,
                     author: AccountId::from(DAVE),
-                    latest_slot_number: 5.into(),
+                    latest_slot_number: 0.into(),
```

Comment:
we dont advance the slots in the integration tests?

Reply:
I believe it's because we are in block 0, let me check what happens if I set the block number to 1.

Reply:
I see there are some tests in which we have a 1. I think it is fair.

Reply:
After `run_to_block(10)`, the slot number is 9, I expected 10 but I guess it doesn't matter?

Reply:
I guess it's a problem with the way we do integration tests, in zombienet the slot number stored in author noting at block 25 matches the slot found in the header of block 25.